### PR TITLE
#154 ability to delete flowcharts

### PR DESCRIPTION
**Pull request by macrivello**
Adding the ability to delete a flowchart.

### CODE REVIEW

File: webapp/app/components/flowchart/flowchart.component.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {Observable} from 'rxjs/Observable';
 })
 export class FlowchartComponent implements OnInit, OnDestroy {
   @Input() flowchartView: FlowchartView;
+  @Output() onDeleteFlowchart = new EventEmitter();
```
Comment (macrivello) [inline webapp/app/components/flowchart/flowchart.component.ts]: FYI, this is actually not used right now.

File: webapp/app/components/flowchart/flowchart.component.ts [hunk 2]
```diff
@@ -45,4 +46,13 @@ export class FlowchartComponent implements OnInit, OnDestroy {
     });
   }
 
+  onFlowchartEdit() {
+    console.log("onFlowchartEdit");
```
Comment (macrivello) [inline webapp/app/components/flowchart/flowchart.component.ts]: This is implemented in another branch.

File: webapp/app/services/flowchart.service.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {FlowchartView} from '../models/flowchart-view.model';
 @Injectable()
 export class FlowchartService {
 
-  private currentFlowchartID: number;
+  private activeFlowchartId: number;
```
Comment (macrivello) [inline webapp/app/services/flowchart.service.ts]: Is this a better name?

File: webapp/app/services/flowchart.service.ts [hunk 4]
```diff
@@ -117,14 +150,14 @@ export class FlowchartService {
   }
 
   // TODO: Make this more clear
-  updateFlowchart() {
-    console.log('updateFlowchart');
-    if (isNullOrUndefined(this.currentFlowchartID)){
+  fetchAndUpdateActiveFlowchart() {
```
Comment (macrivello) [inline webapp/app/services/flowchart.service.ts]: I think the 'fetchAndUpdate' name changes better reflect the functionality

File: webapp/app/services/flowchart.service.ts (hunk 3 above)
Comment (JPautz) [inline webapp/app/services/flowchart.service.ts]: I do like it alot more

File: webapp/app/components/flowchart/flowchart.component.ts (hunk 2 above)
Comment (JPautz) [inline webapp/app/components/flowchart/flowchart.component.ts]: on a different branch?

**Pull request merged**